src/Microsoft.Azure.WebJobs.Extensions.Dapr/Services/DaprServiceClient.cs

```diff
@@ -159,6 +159,7 @@ static async Task ThrowIfDaprFailure(HttpResponseMessage response)
                     JsonSerializer.Serialize(body, JsonUtils.DefaultSerializerOptions),
                     Encoding.UTF8,
                     "application/json");
+                req.Content.Headers.ContentType.CharSet = string.Empty;
```

Comment:
why are we setting it to empty string? for applitcation/json it should anyway be treated as UTF8 encoding. Is this required?

Reply:
The content-type is being set to 'application/json; utf-8' instead of 'application/json' (which changed when we moved from Newtonsoft to STJ), which broke the service invocation. This change fixes that.